CONTRIBUTING.md

```diff
@@ -55,6 +55,28 @@ Contributions are essential for keeping this extension great. We try to keep it
 
     ![ Launch Extension ](./images/LaunchExtension.png)
 
+### Debugging
+
+After completing the prerequisite steps above, and after launching the **Extension Development Host** for `vscode-xml`, the debugger also extends to `lemminx`, and can be used as follows:
+
+#### On VSCode
+
+1. Ensure that the most recent changes of `lemminx` have been built before launching the **Extension Development Host** (as per Step 7 in [Steps](#Steps)), and that the **Extension Development Host** is running.
+
```

Comment:
Add a step between 1 and 2 about opening lemminx in VS Code, and mention that the "Debug (Attach) - Remote" will appear in VS Code when you have lemminx open